### DISCUSSION TITLE
Unexpected token with v 3.6.0?

### DISCUSSION THREAD

**Discussion by jojugaad** [formvuelate/formvuelate#231] [Q&A] [answered]
Hi there,

I've just upgraded my FormVuelate package to v 3.6.0 but am getting this weird "Unexpected token" error now. 

_frontend_1  |  ERROR  Failed to compile with 1 error10:47:59 AM
frontend_1  | 
frontend_1  |  error  in ./node_modules/formvuelate/dist/formvuelate.umd.js
frontend_1  | 
frontend_1  | Module parse failed: Unexpected token (97:38)
frontend_1  | You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
frontend_1  | |   const findElementInSchema = (model, normalizedSchema) => {
frontend_1  | |     const schemaElements = findSchemaElements(normalizedSchema);
frontend_1  | >     const isCorrectElement = el => el?.model === model;
frontend_1  | | 
frontend_1  | |     if (!schemaElements.length) {
frontend_1  |_ 

Line 97 in formvuelate.umd.js reads like this:

`const isCorrectElement = el => el?.model === model;`

Am I the only one with this problem? Any idea how to solve? :)

**Comment by marina-mosti** [comment]
Hey @jojugaad thanks for letting us know, the problem is that we're using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining the optional changing syntax. We're going to push a fix probably later tonight to run this through babel so that it's not a problem in configurations that don't allow it

**Reply by logaretm** [reply]
Converted this to an issue in #232 to track it in commits.

**Comment by marina-mosti** [comment, marked as answer]
Solved on 3.6.1 #233

**Reply by jojugaad** [reply]
Great. Thank you!!
